tests/consts.sh

```diff
@@ -1,5 +1,7 @@
 OASIS_CORE_VERSION=21.1-dev
-# To download a pre-released version of oasis-core built by oasis-core/release-dev action
-# set artifact ID to use here.
-# To download a released version of oasis-core unset this variable.
+# To download a pre-released version of oasis-core built by
+# oasis-core/release-dev action, set the artifact ID to use here.
+# You must set GITHUB_TOKEN as well, if it is unset, the release version
+# specified above will be downloaded.
```

Comment:
is it really correct for the user to choose which version in this way? we used to use these prerelease versions because we needed some features from them, and it would have been incorrect to use the released version

Reply:
By default, the release version given above doesn't exist anyways, so the download will fail.  I imagine that at some point the whole thing will stabilize enough that we will be able to use a released version instead -- it's very annoying to have to provide a github token to run the tests locally (this was my primary motivation for this change).